server/adaptors/integrations/__data__/repository/amazon_vpc_flow/assets/create_skipping_index-1.0.0.sql

```diff
@@ -2,12 +2,11 @@ CREATE SKIPPING INDEX ON {table_name} (
     accountid BLOOM_FILTER,
     region VALUE_SET,
     severity_id VALUE_SET,
-    src_endpoint.ip BLOOM_FILTER,
-    dst_endpoint.ip BLOOM_FILTER,
-    src_endpoint.svc_name VALUE_SET,
-    dst_endpoint.svc_name VALUE_SET,
-    request_processing_time MIN_MAX,
-    traffic.bytes MIN_MAX
+    `src_endpoint.ip` BLOOM_FILTER,
+    `dst_endpoint.ip` BLOOM_FILTER,
+    `src_endpoint.svc_name` VALUE_SET,
+    `dst_endpoint.svc_name` VALUE_SET,
+    `traffic.bytes` MIN_MAX
```

Comment:
Are we removing the field `request_processing_time MIN_MAX,`?

Reply:
It doesn't exist in the data schema, so it wasn't supposed to be there anyways